apps/protocol-api/src/prisma/resolvers/Guild.ts

```diff
@@ -151,28 +199,27 @@ export class GuildCustomResolver {
     return result[0].avg;
   }
 
+  @TypeGraphQL.Query(_returns => VerificationSettings)
+  async getGuildVerificationSettings(
+    @TypeGraphQL.Ctx() { prisma }: Context,
+    @TypeGraphQL.Args() args: GetGuildVerificationSettingsArgs,
+  ) {
+    const verificationSettings = await prisma.verificationSettings.findFirst({
+      where: {
+        guild_id: args.where.guildId,
+      },
+    });
+    return verificationSettings;
+  }
```

Comment:
the resolver generated by prisma uses the id of the verificationSettings row/object. but here we're retrieving the verification settings for a particular guild (guild id). which is not autogenerated

Reply:
Good pooint, though we probably should use id instead to fetch a single setting. The frontend should have the setting id ahead of time because it has the dao.